src/devices/machine/wd2010.cpp

```diff
@@ -580,20 +581,14 @@ void wd2010_device::read_sector(uint8_t data)
 
 			// NO "CRC ERROR"
 
-			// FLAG "M" SET? (MULTIPLE SECTOR TRANSFERS)
-			if (data & 4)
-				logerror("WD2010 (READ): MULTIPLE SECTOR READ (M = 1).\n");
-
-			// Assume: NO "M" (MULTIPLE SECTOR TRANSFERS)
-
 			m_out_bcs_cb(0); // deactivate BCS (!)
 
 			m_out_bcr_cb(0); // strobe BCR
 			m_out_bcr_cb(1);
 
 			// set BDRQ (NOTE: DRQ status bit 3 reflects state of BDRQ)
 			m_status |= STATUS_DRQ;
-			m_out_bdrq_cb(1);
+			m_out_bdrq_cb(0);
```

Comment:
Is this going to affect anything else using this device?

Reply:
The only other wd2010 user i found is the DEC rainbow. I couldn't test this as i have no images and didn't found any image online.